services/brig/src/Brig/Options.hs

```diff
@@ -504,11 +504,11 @@ data Settings = Settings
   }
   deriving (Show, Generic)
 
-defaultUserLocale :: Locale
-defaultUserLocale = Locale (Language EN) Nothing
-
 defaultTemplateLocale :: Locale
-defaultTemplateLocale = defaultUserLocale
+defaultTemplateLocale = Locale (Language EN) Nothing
+
+defaultUserLocale :: Locale
+defaultUserLocale = defaultUserLocale
```

Comment:
cycle.

```suggestion
defaultUserLocale = defaultTemplateLocale
```

(anyway it'd also be ok with me if you made both of them just `en`.)